kernel/src/idt.c

```diff
@@ -24,14 +25,17 @@ void setup_idt() {
         uint64_t base;
     } __attribute__((packed)) idt = {.size = sizeof(g_idt) - 1, .base = (uint64_t)&g_idt};
 
+    // initialize controllers for hardware interrupts
+    setup_interrupt_requests();
+
     asm(
         // Set IDT
         "lidt (%[idt])\n"
         // Enable interrupts
         "sti\n"
         : // No output
           // Input
-        : [idt] "r"(&idt));
+        : [ idt ] "r"(&idt));
```

Comment:
What is this change=

Reply:
this was caused by clang-format.

Reply:
This is out of line whith all other places in the code, could you remove this change